appveyor.yml

```diff
@@ -36,10 +36,10 @@ build:
 build_script:
     - cd "%APPVEYOR_BUILD_FOLDER%"\src
     - if "%PlatformToolset%"=="v140" msbuild CSScriptNpp.sln  /p:configuration="%configuration%" /p:platform="%platform_input%" /p:PlatformToolset="%PlatformToolset%" /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
-    - if "%PlatformToolset%"=="v141" msbuild CSScriptNpp.2017.sln  /p:configuration="%configuration%" /p:platform="%platform_input%" /p:PlatformToolset="%PlatformToolset%" /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
+    - if "%PlatformToolset%"=="v141" msbuild CSScriptNpp.2017.sln /m /p:configuration="%configuration%" /p:platform="%platform_input%" /p:PlatformToolset="%PlatformToolset%" /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
 
 after_build:
-    - cd "%APPVEYOR_BUILD_FOLDER%"\CSScriptNpp
+    - cd "%APPVEYOR_BUILD_FOLDER%"\src\CSScriptNpp
```

Comment:
@oleg-shilo To run the tests for v141 on appveyor this fix is needed. For v140 I did an update of all the nuget packages, see https://github.com/chcg/cs-script.npp/commits/nuget_updates, but probably that is overkill and just one or two steps are really the relevant ones.

Reply:
Yeah I noticed "after_build" problems but didn't want to change it without discussing it with you.  

I will merge your last change. Not a problem. But keep in mind that appveyor for this project is great as a codebase health check but not as a distro aggregateor. You probably already noticed this. The distro package needs to be composed manually and with an extreme care as it need to include the whole Roslyn. 

Though that migration to Syntaxer will also make distro authoring much simpler as Syntaxer.exe will have the whole Roslyn embedded.

Thank you,
Oleg

Reply:
Please synch your branch with the master. I just got it right:

![image](https://user-images.githubusercontent.com/16729806/33112135-bb8dbbf6-cfa5-11e7-8294-ea37393f884d.png)

Reply:
Agree. I just didn't want to remove completely any deployment content as I may want to re-enable it in the future. In the latest commit I just commented it out.

As for VS2015, my very quick test indicated that there are more problems than just NuGet. Though I didn't have time to have a closer look so I can be wrong. I just decided not to invest in that IDE as VS2017 is the chosen as a primary one. Though will need to remove it from the sources.

And x64. This is tricky :) Notepad++ does not support x64 plugins. Hold on, I know there is x64 N++, but... It is raw with respect to plugin support. Last time I checked (~3 months ago) it didn't  even have PluginManager. All plugins supposed to be installed manually. Even more... The plugin code needs to be rewritten to support x64!!!! Thus I just don't see N++ being ready for x64. Not technically but product wise.

I actually wrote about it in this article: https://www.codeproject.com/Articles/1213718/Authoring-plugins-for-VSCode-Sublime-Text-and

-----

On a different note, I just wanted to thank you for bringing AppVeyor support. It was something that I always wanted but didn't have time for.

Reply:
ok

For VS2015 I just checked that the build is working again, not that the output is functional.

Regarding x64:
There is a PM version for x64, see https://github.com/bruderstein/nppPluginManager/releases/tag/v1.4.9.
But PM is no longer bundled with N++, see https://github.com/bruderstein/nppPluginManager/issues/80 and https://notepad-plus-plus.org/community/topic/14315/notepad-7-5-release.
There is some development ongoing in N++ itself for a replacement. Until that is available unfortunately manual installation of PM is needed. There is also a PM list for x64 ready plugins, see https://github.com/bruderstein/npp-plugins-x64/blob/master/plugins/plugins64.xml.
So if a x64 version is of interest I could check what is necessary therefore.

Reply:
I had another look at VS2015 solution and indeed it was very manor problem there.  I had to remove an obsolete project (Formatter) and all is good now:

![image](https://user-images.githubusercontent.com/16729806/33255009-a65beebc-d39f-11e7-999e-ff7ffed41783.png)

>So if a x64 version is of interest I could check what is necessary therefore

Thanks. I have already tested an alternative .NET [host-adapter](https://github.com/kbilsted/NotepadPlusPlusPluginPack.Net) and it does work. But it has slightly different interface so refitting it to the existing project is needed before the plugin can go x64. And even after that I may need to maintain two separate codebases for x86 and x64. I hope I am wrong about this :)

That's why I want to try to migrate to Syntaxer before making any x64 steps. Though the reasoning is not technical but rather logistical. 

But with Christmas quickly approaching and if I don't conduct Syntaxer integration within a week then probably it will be pushed to the next year.

Reply:
Regarding x64 it should be possible to have just one common code base for it and just the visual studio configuration needs to be "duplicated" to support x64. It is most probably also possible to just merge to x64 relevant code back into the existing files. The additional NotepadPlusPlusPluginPack.Net changes mostly implement a more structured access to the existing interfaces.

Reply:
Yes. It is possible. Though it will require some changes in the plugin implementation as the new host-adapter has some API differences (not only structs). At least my very ambitious attempt to just build the full x64 plugin dramatically failed. :)

Nevertheless, I have already mentally committed to x64 and the require _syntaxer migration_ is now in progress. Just got auto-completion working. 

Next will be the actual x64 migration.

Reply:
OK. The latest commit 7338d049ce1444a2508e83d182eb567ee68aa4ac was the last bit for Syntaxer migration. 

I will be away till the end of years so I don't have time to make the release before 2018. But the code base is OK and the binaries are already available: https://github.com/oleg-shilo/cs-script.npp/tree/master/bin/Plugins   

During Syntaxer migration I actually managed to bring your Roslyn packages updates. I am quite happy with the outcome. The distro became very compact and simple. There is no need to host Roslyn under ProgramFiles - simple plugin upgrade. This also lets packaging to be done on AppVeyor. Some, cherry on top. :)

The x64 effort can now begin. But... now i will have some break. 
Merry Christmas!